### PR TITLE
Close unclosed tags and remove extra markups

### LINKED ISSUE #146: Missing </div> tag
There is a `</div>` tag missing at the very end of the HTML file.

### PR DESCRIPTION
**Pull request by Nalinc**
Fixes #146 and #145

### CODE REVIEW

**Comment by Nalinc** [comment]
@samuelgoto  @redblobgames Can you please review this PR?

**Comment by redblobgames** [comment]
@Nalinc There's no one actively coding on this project until the summer (google summer of code) so my guess is that it'll be handled then.

**Comment by norvig** [comment]
There's a conflict now.

**Comment by Nalinc** [comment]
Resolved conflicts with the base branch from original PR.

File: 2-Intelligent-Agents/index.html [hunk 1]
```diff
@@ -139,7 +140,7 @@ <h3>Evaluating multiple agents</h3>
           <h3>Evaluating many data sets</h3>
 
           TODO: one agent on many different data sequences (avoid overfitting)
-
+        </div>
```
Comment (redblobgames) [inline 2-Intelligent-Agents/index.html]: This </div> seems to be too many. Can you check this? It looks like you have 6 <div> and 7 </div> in this file.

Comment (Nalinc) [reply 2-Intelligent-Agents/index.html]: @redblobgames fixed it

**Comment by redblobgames** [comment]
Thank you!

**Pull request merged**